gslib/commands/signurl.py

```diff
@@ -99,9 +89,9 @@
   for the specified HTTP method and valid for the given duration.
 
   NOTE: Unlike the gsutil ls command, the signurl command does not support
-  operations on sub-directories. For example, if you run the command:
-
-    gsutil signurl <private-key-file> gs://some-bucket/some-object/
+  operations on sub-directories. For example, unless you have an object named
+  `some-directory/` stored inside the bucket `some-bucket`, the following
```

Comment:
Lines 93-94: code formatting should be double back ticks instead of single (not sure why, but it's a nuance of gsutil formatting)

Reply:
@karenarialin Thanks. I assumed that it uses Markdown formatting, but looks like rest syntax.